src/test/java/com/google/devtools/build/lib/skyframe/SequencedSkyframeExecutorTest.java

```diff
@@ -623,7 +623,8 @@ public Path getPathFromSourceExecPath(Path execRoot, PathFragment execPath) {
           },
           new ActionKeyContext(),
           Predicates.<Action>alwaysTrue(),
-          null);
+          null,
+          PathFragment.create("bin"));
```

Comment:
nitnit: It doesn't matter, but typically that should be something like 'bazel-out` -- may make tests easier to read if we use a common one or `out` at least.